### PR TITLE
Chore: Bump code coverage to 100% again

### PR DESCRIPTION
**Pull request by sansan**
### Description

add missing cases to get coverage to 100%

image

A couple of weird ones that I didn't fix (might be worth adding ignore to these):
image

image

### Breaking Changes

none

### JIRA Link

https://polymesh.atlassian.net/browse/DA-552

### Checklist

- [x] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/base/__tests__/utils.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { TypeDef } from '@polkadot/types/types';
+
+import { TransactionArgumentType } from '~/types';
+
+import { processType } from '../utils';
+
+describe('Process Type', () => {
+  it('should be a function', () => {
+    expect(typeof processType).toBe('function');
```
Comment (polymath-eric) [inline src/base/__tests__/utils.ts:9]:
```suggestion
    expect(processType).toBeInstanceOf(Function)
```

(nit) I think built ins are a bit more readable

**Review by polymath-eric — APPROVED** [review]

**Comment by polymath-eric** [comment]
> might be worth adding ignore to these

Can you add ignore to them? I think the goal is `yarn test` should be all green so its easy to spot when something is missed

**Review by polymath-eric — DISMISSED** [review]
I ran them now. Please add the ignore comment for the weird cases

**Review by prashantasdeveloper — APPROVED** [review]

**Pull request merged**